crates/sui-types/src/execution.rs

```diff
@@ -170,6 +171,12 @@ impl ExecutionResultsV2 {
     }
 }
 
+pub enum ExecutionTiming {
+    Success(Duration),
+    Abort(Duration),
```

Comment:
does `Success` and `Abort` not already duplicate information contained in effects? maybe this is the most expedient way, but at least wanted to ask if there's a way to not duplicate that here

Reply:
I don't think it is easy to figure out which command aborted by looking at the effects, although it may be possible.